configure.ac

```diff
@@ -152,12 +152,17 @@ if test "x$GCC_ATOMIC_BUILTINS_NEED_LIBATOMIC" = xyes; then
     LDFLAGS="-latomic $LDFLAGS"
 fi
 
+# Running the functional tests without building Nix is useful for testing
+# different pre-built versions of Nix against each other.
+AC_ARG_ENABLE(build, AS_HELP_STRING([--disable-build],[Do not build nix]),
+  DO_BUILD=$enableval, DO_BUILD=yes)
+AC_SUBST(DO_BUILD)
 # Building without tests is useful for bootstrapping with a smaller footprint
 # or running the tests in a separate derivation. Otherwise, we do compile and
 # run them.
 AC_ARG_ENABLE(tests, AS_HELP_STRING([--disable-tests],[Do not build the tests]),
-  tests=$enableval, tests=yes)
-AC_SUBST(tests)
+  DO_TESTS=$enableval, DO_TESTS=yes)
+AC_SUBST(DO_TESTS)
```

Comment:
This is good, but `ENABLE_` has some precedent here unlike `DO_`.
Any reason not to choose that?

Reply:
Nope, that's definitely better!